### PR TITLE
OSOE-548: Upgrade to Orchard Core 1.6 in UI-Testing-Toolbox

### PR DESCRIPTION
**Pull request by Psichorex**
[OSOE-548](https://lombiq.atlassian.net/browse/OSOE-548)

[OSOE-548]: https://lombiq.atlassian.net/browse/OSOE-548?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Lombiq.Tests.UI/Extensions/TenantsUITestContextExtensions.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ public static class TenantsUITestContextExtensions
 
         if (!string.IsNullOrEmpty(featureProfile))
         {
-            await context.SetDropdownByTextAsync("FeatureProfile", featureProfile);
+            await context.ClickReliablyOnAsync(By.XPath("//option[@value='Features Guard']"));
```
Comment (Piedone) [inline Lombiq.Tests.UI/Extensions/TenantsUITestContextExtensions.cs]: This extension method is about creating a tenant in Orchard, and is independent of any module of ours. It shouldn't include a reference to Features Guard.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.Tests.UI/Extensions/TenantsUITestContextExtensions.cs (hunk 1 above)
Comment (Psichorex) [reply Lombiq.Tests.UI/Extensions/TenantsUITestContextExtensions.cs]: Thank you it is a good catch I addressed it be making it flexible.

File: Lombiq.Tests.UI/Extensions/TenantsUITestContextExtensions.cs [hunk 2]
```diff
@@ -49,7 +49,7 @@ public static class TenantsUITestContextExtensions
 
         if (!string.IsNullOrEmpty(featureProfile))
         {
-            await context.SetDropdownByTextAsync("FeatureProfile", featureProfile);
+            await context.ClickReliablyOnAsync(By.XPath($"//option[@value='{featureProfile}']"));
```
Comment (Piedone) [inline Lombiq.Tests.UI/Extensions/TenantsUITestContextExtensions.cs:52]: Why can't `SetDropdownByTextAsync` be used?

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.Tests.UI/Extensions/TenantsUITestContextExtensions.cs (hunk 2 above)
Comment (Psichorex) [reply Lombiq.Tests.UI/Extensions/TenantsUITestContextExtensions.cs:52]: The editor has changed seemingly it is no longer a dropdown. It's a list where you have to click and choose the item.

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]

File: Lombiq.Tests.UI/Lombiq.Tests.UI.csproj [hunk 3]
```diff
@@ -89,15 +89,18 @@
...
     <PackageReference Include="Lombiq.HelpfulLibraries.RestEase" Version="5.1.2" />
     <PackageReference Include="Lombiq.Npm.Targets" Version="1.3.1" />
+    <!--The reference for NodeJs Extensions is for the reason to prevent NuGet build 
+    to use a PNPM version higher than 7.x.x -->
+    <PackageReference Include="Lombiq.NodeJs.Extensions" Version="1.2.3" />
   </ItemGroup>
 
   <!-- These are necessary for symbols NuGet packaging, otherwise Shouldly would prevent PDBs to be packaged, see:
-  https://github.com/NuGet/Home/discussions/11541. -->
+  https://github.com/NuGet/Home/dissussions/11541. -->
```
Comment (Piedone) [inline Lombiq.Tests.UI/Lombiq.Tests.UI.csproj]:
```suggestion
  https://github.com/NuGet/Home/discussions/11541. -->
```

File: Lombiq.Tests.UI/Lombiq.Tests.UI.csproj [hunk 4]
```diff
@@ -111,6 +114,6 @@
     </EmbeddedResource>
     <None Remove="node_modules\**" />
   </ItemGroup>
-
+
```
Comment (Piedone) [inline Lombiq.Tests.UI/Lombiq.Tests.UI.csproj]: There's unneeded whitespace both before and after this block.

**Review by Piedone — CHANGES_REQUESTED** [review]

**Review by Piedone — APPROVED** [review]

File: Lombiq.Tests.UI/Models/VisualVerificationMatchApprovedContext.cs [hunk 5]
```diff
@@ -33,6 +33,8 @@ public class VisualVerificationMatchApprovedContext
 
         BaselineImageFileName = configuration.BaselineFileNameFormatter(configuration, this);
         BaselineImageResourceName = $"{testFrame.MethodInfo.DeclaringType!.Namespace}.{BaselineImageFileName}.png";
+        var testDirectory = new FileInfo(testFrame.GetFileName());
+        var testPath = testDirectory.Directory.FullName;
```
Comment (Piedone) [inline Lombiq.Tests.UI/Models/VisualVerificationMatchApprovedContext.cs]: These are not needed but Jocó removed them.

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]
Just marking the merge from `issue/OSOE-641`.

**Review by Piedone — APPROVED** [review]

**Pull request merged**